### PR TITLE
LPD-33706 Use of library with known vulnerability: cxf 3.4.10

### PR DESCRIPTION
**Pull request by jgarcialr**
[LPD-33706](https://liferay.atlassian.net/browse/LPD-33706)

We've chosen to upgrade to cxf 3.5.9 instead of 3.5.8.
The reason is org.apache.cxf:cxf-rt-rs-security-jose has a security vulnerability in v3.5.8 that is solved in v3.5.9.
More info in: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-32007

### CODE REVIEW

**Comment by jgarcialr** [comment]
ci:test:relevant

**Comment by jgarcialr** [comment]
ci:test:security

**Comment by jgarcialr** [comment]
ci:test:sf

File: source-formatter.properties [hunk 1]
```diff
@@ -618,27 +618,27 @@
...
-        org.apache.cxf:cxf-rt-rs-security-oauth2:3.4.10,\
-        org.apache.cxf:cxf-rt-rs-sse:3.4.10,\
-        org.apache.cxf:cxf-rt-security:3.4.10,\
-        org.apache.cxf:cxf-rt-transports-http:3.4.10,\
-        org.apache.cxf:cxf-rt-ws-addr:3.4.10,\
-        org.apache.cxf:cxf-rt-ws-policy:3.4.10,\
-        org.apache.cxf:cxf-rt-wsdl:3.4.10,\
-        org.apache.cxf:cxf-tools-common:3.4.10,\
-        org.apache.cxf:cxf-tools-validator:3.4.10,\
+        org.apache.cxf:cxf-core:3.5.9,\
```
Comment (jeffwu0724) [inline source-formatter.properties:621]: Hi @jgarcialr ,

I noticed some tiny typo in the commit message, we may need to upgrade it to "LPD-33706 upgrade org.apache.cxf dependencies to 3.5.9 in source-formatter.properties"

Thanks,
Jeff

**Comment by marianoalvarosaiz** [comment]
Rebased and fixed commit message in https://github.com/liferay-appsec/liferay-portal/pull/1914

**Comment by marianoalvarosaiz** [comment]
ci:close

**Pull request closed without merging**